omero/sysadmins/limitations.rst

```diff
@@ -79,6 +85,20 @@ This primarily affects the following file formats:
 This issue can be avoided by pre-generating pyramidal OME-TIFF images as
```

Comment:
I would now move this sentence at the end of the section and reformulate it as

```
It is recommended to pre-generate pyramidal OME-TIFF images as described above and to set `omero.pixeldata.max_plane_float_override` to `False`  server-side.
```

Reply:
fixed in https://github.com/ome/omero-documentation/pull/2256/commits/682e4ccebe02526b99011859bac8453a71c64379 and https://github.com/ome/omero-documentation/pull/2256/commits/776a07d4b60c84cb039a71c789524b1970b65287